spec/requests/api/v1/expertises_spec.rb

```diff
@@ -0,0 +1,137 @@
+require 'swagger_helper'
+
+RSpec.describe 'api/v1/expertises', type: :request do
+
+  path '/api/v1/users/{user_id}/expertises' do
+    parameter name: 'user_id', in: :path, type: :string, description: 'user_id'
```

Comment:
Suggest we update the description to e.g. "The ID of user that needs to be fetched" for better understanding, and global change to other 'user_id' accordingly